cameo/strain_design/heuristic/evolutionary/optimization.py

```diff
@@ -358,12 +362,42 @@ def _calculate_fitness(self, solution, decoded):
             return self.objective_function(self.model, solution, decoded)
 
 
+class SwapEvaluator(KnockoutEvaluator):
```

Comment:
I think we can have a generic Evaluator. I will be conceptually easier and later on we can extend evaluators that are not Knockout related.